test/AssemblySet_basic_test.py

```diff
@@ -55,21 +56,20 @@ def setUpClass(cls):
         # copy test file to scratch area
         fna_filename = "seq.fna"
         fna_path = os.path.join(cls.cfg['scratch'], fna_filename)
-        shutil.copy(os.path.join("data", fna_filename), fna_path)
+        shutil.copy(os.path.join(TEST_BASE_DIR, "data", fna_filename), fna_path)
```

Comment:
make sure that the full path is available, instead of having a relative path that depends on where the tests are called from